common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1746,6 +1746,34 @@
     ],
     "sqlState" : "22012"
   },
+  "INVALID_AGGREGATE_FILTER" : {
+    "message" : [
+      "The FILTER expression <filterExpr> in an aggregate function is invalid."
```

Comment:
To-do: for clarity, surround all <*Expr> with quotes?

Reply:
Not needed now since I changed from `toPrettySQL()` to `toSQLExpr()` which basically adds quotes around the SQL string.